Assets/Project/Modules/Enemies/Turret/Scripts/ParabolicTurret/ParabolicProjectile.cs

```diff
@@ -0,0 +1,194 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Cysharp.Threading.Tasks;
+using Popeye.Core.Pool;
+using Popeye.Core.Services.ServiceLocator;
+using Popeye.Modules.CombatSystem;
+using Unity.Mathematics;
+using UnityEngine;
+
+public class ParabolicProjectile : RecyclableObject
```

Comment:
This class is doing too many things. First you should split:
- ParabolicProjectile class (manager)
- ParabolicProjectileBodyView class
- ParabolicProjectileLineView class
- ParabolicProjectilePathComputer class
- ParabolicProjectileMovement class